storage/src/trie_hash.rs

```diff
@@ -8,7 +8,7 @@ use sha2::digest::{generic_array::GenericArray, typenum};
 
 /// A hash value inside a merkle trie
 /// We use the same type as returned by sha2 here to avoid copies
-#[derive(PartialEq, Eq, Clone, Default)]
+#[derive(PartialEq, Eq, Clone, Default, Copy)]
```

Comment:
I don't think we want this. Hashes are big, and I'd rather be explicit about where they are being copied rather than telling the compiler we don't care about the cost of copying hashes.